.cicd/create-docker-from-binary.sh

```diff
@@ -1,53 +1,66 @@
 #!/bin/bash
 echo '--- :evergreen_tree: Configuring Environment'
 set -euo pipefail
-
+. ./.cicd/helpers/sanitize.sh
 buildkite-agent artifact download '*.deb' --step ':ubuntu: Ubuntu 18.04 - Package Builder' .
-echo ":done: download successful"
-
-SANITIZED_BRANCH=$(echo "$BUILDKITE_BRANCH" | sed 's.^/..' | sed 's/[:/]/_/g')
-SANITIZED_TAG=$(echo "$BUILDKITE_TAG" | sed 's.^/..' | tr '/' '_')
-echo "$SANITIZED_BRANCH"
-echo "$SANITIZED_TAG"
-
-# do docker build
-echo '+++ :docker: Building Container'
-echo ":docker::build: Building image..."
+SANITIZED_BRANCH="$(sanitize "$BUILDKITE_BRANCH")"
+echo "Branch '$BUILDKITE_BRANCH' sanitized as '$SANITIZED_BRANCH'."
+SANITIZED_TAG="$(sanitize "$BUILDKITE_TAG")"
+[[ -z "$SANITIZED_TAG" ]] || echo "Branch '$BUILDKITE_TAG' sanitized as '$SANITIZED_TAG'."
+# docker build
+echo '+++ :docker: Build Docker Container'
 DOCKERHUB_REGISTRY="docker.io/eosio/eosio.cdt"
-
-BUILD_TAG=${BUILDKITE_BUILD_NUMBER:-latest}
-DOCKER_BUILD_GEN="docker build -t eosio_cdt_image:$BUILD_TAG -f ./docker/dockerfile ."
-echo "$ $DOCKER_BUILD_GEN"
-eval $DOCKER_BUILD_GEN
-
-#tag and push on each destination AWS & DOCKERHUB
-echo '+++ :arrow_up: Pushing Container'
+IMAGE="${DOCKERHUB_REGISTRY}:${BUILDKITE_COMMIT:-latest}"
+DOCKER_BUILD="docker build -t '$IMAGE' -f ./docker/dockerfile ."
+echo "$ $DOCKER_BUILD"
+eval $DOCKER_BUILD
+# docker tag
+echo '--- :label: Tag Container'
 EOSIO_REGS=("$EOSIO_CDT_REGISTRY" "$DOCKERHUB_REGISTRY")
-for REG in ${EOSIO_REGS[@]}; do
-    DOCKER_TAG_COMMIT="docker tag eosio_cdt_image:$BUILD_TAG $REG:$BUILDKITE_COMMIT"
-    DOCKER_TAG_BRANCH="docker tag eosio_cdt_image:$BUILD_TAG $REG:$SANITIZED_BRANCH"
-    echo -e "$ Tagging Images: \n$DOCKER_TAG_COMMIT \n$DOCKER_TAG_BRANCH"
-    eval $DOCKER_TAG_COMMIT 
+for REG in ${REGISTRIES[@]}; do
```

Comment:
Should `REGISTRIES` be `EOSIO_REGS`?

Reply:
Yes it should....that's what I get for copying and pasting.

Reply:
Fixed on all three pull requests. Thank you!